src/popup/snooze.scss

```diff
@@ -1,34 +1,80 @@
 @import 'node_modules/rc-calendar/assets/index';
 @import 'node_modules/rc-time-picker/assets/index';
 
-$panel-width: 250px;
+$panel-width: 320px;
+$panel-height: 474px;
+
+@font-face {
+  font-family: 'Fira Sans';
+  src: url('FiraSans-Regular.woff') format('woff');
+  font-weight: 400;
+  font-style: normal;
+}
 
 html,
 body {
   margin: 0;
   padding: 0;
   box-sizing: border-box;
-  font-family: 'Fira Sans', -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Droid Sans', 'Helvetica Neue', sans-serif;
-  font-size: 14px;
+  font-family: 'Fira Sans';
+  font-size: 16px;
+  color: #000000;
+  background-color: #333333;
+  letter-spacing: -.69px;
   width: $panel-width;
+  height: $panel-height;
   cursor: default;
   overflow: hidden;
 }
 
 body {
-  margin: 0 0 50px 0;
+  // margin: 0 0 50px 0;
+  margin: 0;
   -moz-user-select: none;
   user-select: none;
 }
 
+.panel-wrapper {
+  width: 100%;
+  height: 100%;
+  background-color: #333333;
```

Comment:
Maybe we should put the `background-color` on the `body`, so that the arrow will pick it up…